### ISSUE TITLE
Request a feature

### ISSUE THREAD

**Issue by M4x28** [cms-multi/filament-clear-cache#3]
Sorry if I'm writing in the bug section but the other section doesn't work.
I am writing to suggest adding the button to execute the 'php artisan route:cache' command or at least suggest in the documentation a way to implement it with the clear cache button.

**Comment by howdu** [comment]
You can add it as a custom command: https://github.com/cms-multi/filament-clear-cache#customizing

```php
use CmsMulti\FilamentClearCache\Facades\FilamentClearCache;

public function boot()
{
    FilamentClearCache::addCommand('route:cache');
}
```

**Comment by howdu** [comment]
Fixed in version 1.0.3 - https://github.com/cms-multi/filament-clear-cache/releases/tag/1.0.3
